### PR TITLE
Added additional protected region (pythonHL)

### PR DESCRIPTION
**Pull request by KrystianKedron**
Added PR:
- above the tango import
- after the commands and before the main() method

To fix the issue: #89

### CODE REVIEW

File: fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend [hunk 1]
```diff
@@ -462,12 +464,16 @@ def dyn_attr(self, dev_list):
 # Run server
 # ----------
```
Comment (ajoubertza) [inline fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend:465]: Shouldn't this comment should be just before the `def main()` - that is the part that is running the server?

Also, the other usages of `cls.protectedAreaHL` are conditional, checking if the "protected regions" option is enabled.

Maybe something like:

```python
«IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»«cls.protectedAreaHL("custom_functions")»«ENDIF»

# ----------
# Run server
# ----------

def main(args=None, **kwargs):
    """Main function of the «cls.name» module."""
    «IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»«cls.protectedAreaHL("main", "return run((" + cls.name + ",), args=args, **kwargs)", false)»«ELSE»
    return run((«cls.name»,), args=args, **kwargs)«ENDIF»


if __name__ == '__main__':
    main()
```

File: fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend [hunk 2]
```diff
@@ -204,6 +204,8 @@ class «cls.name»(«cls.inheritedPythonClassNameHL»):
 «cls.description.description»
 """
 
+«cls.protectedAreaHL("system_imports")»
```
Comment (ajoubertza) [inline fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend:207]: We need to add a check for "protected regions" enabled here too.

**Review by ajoubertza — COMMENTED** [review]
Thanks for the improvements, some small suggestions.

File: fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend (hunk 2 above)
Comment (KrystianKedron) [reply fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend]: Thx, good hint. Done!

Comment (dlacoste-esrf) [reply fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend]: Can we factorize the check in the protectedAreaHL method ?

Comment (ajoubertza) [reply fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend]: @dlacoste-esrf Do you mean can we move the check for "protected regions enabled" inside `protectedAreaHL`?  That sounds like good idea, if it possible.

Comment (dlacoste-esrf) [reply fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend]: exactly.

**Review by dlacoste-esrf — APPROVED** [review]

File: fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend (hunk 2 above)
Comment (dlacoste-esrf) [reply fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend]: FYI, I merged and pushed the factorized version directly on master.

Comment (KrystianKedron) [reply fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend]: @dlacoste-esrf  thanks for your improvements, looks nice.

**Pull request merged**